.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on:
+  push:
+    branches: [ release-1.15-intel ]
+  pull_request:
+    branches: [ release-1.15-intel ]
+
+jobs:
+  build:
+    # github-hosted runner
+    runs-on: ubuntu-20.04
+    # self-hosted runner
+    # runs-on: [self-hosted]
+    env:
+      DOCKER_TARGETS: docker.pilot docker.istioctl docker.operator docker.install-cni
+      branch_name: release-1.15-intel
```

Comment:
Where do you need this variable? The action/checkout will checkout the correct branch and you have the branch info in GITHUB_ENV.

Reply:
It would be used in proxyv2 build, but now just remove it.